src/main/java/frc/robot/subsystems/ball/shooter/ShooterImpl.java

```diff
@@ -15,6 +16,8 @@
     WPI_TalonSRX shooterController2;
 
     DoubleSolenoid hatch;
+    
+    boolean shooting;
```

Comment:
This should be `volatile`, since some other Subsystem thread (or the main Robot thread) will call the `run/stopShooter` methods to set its value, and this Subsystem's thread will read it in `task`